streams/src/main/java/org/apache/kafka/streams/kstream/internals/KStreamImpl.java

```diff
@@ -356,37 +382,54 @@ public void print(final Printed<K, V> printed) {
     public KStream<K, V> through(final String topic, final Produced<K, V> produced) {
         final ProducedInternal<K, V> producedInternal = new ProducedInternal<>(produced);
         to(topic, producedInternal);
-        return builder.stream(Collections.singleton(topic),
-                              new ConsumedInternal<>(producedInternal.keySerde(),
-                                                     producedInternal.valueSerde(),
-                                                     new FailOnInvalidTimestamp(),
-                                                     null));
+        return builder.stream(
+            Collections.singleton(topic),
+            new ConsumedInternal<>(
+                producedInternal.keySerde(),
+                producedInternal.valueSerde(),
+                new FailOnInvalidTimestamp(),
+                null
+            )
+        );
     }
 
     @Override
     public void foreach(final ForeachAction<? super K, ? super V> action) {
         Objects.requireNonNull(action, "action can't be null");
-        String name = builder.newProcessorName(FOREACH_NAME);
+        final String name = builder.newProcessorName(FOREACH_NAME);
 
-        ProcessorParameters processorParameters = new ProcessorParameters<>(new KStreamPeek<>(action, false), name);
+        final ProcessorParameters processorParameters = new ProcessorParameters<>(
```

Comment:
nit: is it necessary for triggering a warning? My personal preference is that if a single line is not too long, then it is okay to do so.

Reply:
ditto below.